.github/workflows/build-docs.yml

```diff
@@ -27,4 +27,6 @@ jobs:
           echo "SOURCE_BRANCH=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
           echo "SOURCE_TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
       - name: Build docs website
-        run: make build-docs-website
\ No newline at end of file
+        run: |
+          echo "GIT_PYTHON_REFRESH=quiet"
```

Comment:
what is this for?

Reply:
There was a failing build on docs and some research suggested to use it coz of some git behaviour changes